plugins/bitrot/rest.go

```diff
@@ -0,0 +1,189 @@
+package bitrot
+
+import (
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/pborman/uuid"
+)
+
+func bitrotHelpHandler(w http.ResponseWriter, r *http.Request) {
+	// Implement the help logic and send response back as below
+	ctx := r.Context()
+	restutils.SendHTTPResponse(ctx, w, http.StatusOK, "Bitrot Help")
+}
+
+func bitrotEnableHandler(w http.ResponseWriter, r *http.Request) {
+	// Collect inputs from URL
+	p := mux.Vars(r)
+	volName := p["volname"]
+
+	ctx := r.Context()
+	logger := gdctx.GetReqLogger(ctx)
+
+	// Validate volume existence
+	volinfo, err := volume.GetVolume(volName)
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusNotFound, errors.ErrVolNotFound.Error(), api.ErrCodeDefault)
+		return
+	}
+
+	// Check if volume is started
+	if volinfo.State != volume.VolStarted {
+		restutils.SendHTTPError(ctx, w, http.StatusBadRequest, errors.ErrVolNotStarted.Error(), api.ErrCodeDefault)
+		return
+	}
+
+	// Check if bitrot is already enabled
+	if volinfo.BitrotEnabled {
```

Comment:
Can be changed to check volinfo.Options[VkeyFeaturesBitrot]

    val, ok := volinfo.Options[VkeyFeaturesBitrot]
    if ok && val == "on" {